### ISSUE TITLE
Set swipeable prop on Toast component

### ISSUE THREAD

**Issue by hichemfantar** [calintamas/react-native-toast-message#542]
**Is your feature request related to a problem? Please describe.**
Unable to set swipeable default value

**Describe the solution you'd like**
set swipeable prop on Toast component

**Describe alternatives you've considered**

**Additional context**
```
		<Toast
			config={toastConfig}
			position={position}
			autoHide
			swipeable={false}
		/>

```

**Comment by amerllica** [comment]
Yeah, I encountered that. The main component hasn't got the `swipeable` prop type.

**Comment by hichemfantar** [comment]
It seems that it is possible to set the swipeable prop https://github.com/calintamas/react-native-toast-message/blob/main/docs/api.md#props

I don't know remember why i created this issue.
